ab3d2_source/screensetup.s

```diff
@@ -286,6 +286,11 @@ MainScreenTags	dc.l	SA_Width,320
 				dc.l	SA_BitMap,MainBitmap0
 				dc.l	SA_Type,CUSTOMSCREEN
 				dc.l	SA_Quiet,1
+				IFNE	SCREEN_TITLEBAR_HACK
+				dc.l	SA_ShowTitle,0
+				ELSE
+				dc.l	SA_ShowTitle,1
+				ENDC
```

Comment:
I don't think we need to put this into a conditional. I think this will stick around :-)